service/config-service/src/main/java/uk/gov/gchq/palisade/config/service/ConfigUtils.java

```diff
@@ -48,6 +48,8 @@ private ConfigUtils() {
     public static String retrieveConfigurationPath() {
         String path = System.getenv(ConfigConsts.CONFIG_SERVICE_PATH);
         if (isNull(path)) {
+            path = System.getProperty(ConfigConsts.CONFIG_SERVICE_PATH);
+        } else if (isNull(path)) {
```

Comment:
See above comments, if it is possible to set environment variables in failsafe and Surefire, then these two lines can be removed.

Reply:
made change and committed to branch